### PR TITLE
fix: nick command

### PR DESCRIPTION
**Pull request by ghost**
Fixed stuff regarding !nick command.

### CODE REVIEW

**Comment by poke5352** [comment]
Poke Approved :)

File: bot/cogs/utility/nickrequest.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def __init__(self, bot: Bot):
     @commands.command(name="nick")
     # @commands.cooldown(1, command_timeout, commands.BucketType.user)
     async def nick(self, ctx: commands.Context, *, nick):
-        acc_channel = self.bot.get_channel(accept_channel_id)
+        acc_channel = self.bot.get_channel(754818272528040026)
```
Comment (vcokltfre) [inline bot/cogs/utility/nickrequest.py]: Why is this necessary, it's better to have this in a config file not hardcoded

Comment (ghost) [reply bot/cogs/utility/nickrequest.py]: Resolved (:

**Review by vcokltfre — APPROVED** [review]
